config.go

```diff
@@ -95,15 +95,29 @@ type Config struct {
 
 	// File logger options
 	FileLogger *FileLoggerConfig `mapstructure:"file_logger_options"`
+
+	// UseLocalTimeEncoder is used to set the encoder to use local time instead of UTC.
+	UseLocalTimeEncoder bool `mapstructure:"use_local_time_encoder"`
```

Comment:
I'll think about this, but atm, this name is too long, I suggest to use smt like: `use_local_time`.